src/ami/Assignment_4/NumberGame.java

```diff
@@ -0,0 +1,51 @@
+package ami.Assignment_4;
+
+/*
+a. findMaxNumber()
+method should take 3 parameters and print the maximum number from given 3 numbers.
+if all numbers are same print "Maximum number could not found, All given numbers are equals".
+
+b. findMinNumber()
+method should take 3 parameters and print the minimum number from given 3 numbers.
+if all numbers are same print "Minimum number could not found, All given numbers equal numbers".
+
+ */
+
+public class NumberGame {
+	int A;
+	int B;
+	int C;
+	
+	void findMaxNumber(int a , int b, int c){
+		A = a;
+		B = b;
+		C = c;
+		
+		if( a > b && a > c ) {
+			System.out.println("maximum number is : " + a);
+		}else if(b > a && b > c) {
```

Comment:
You already compare in if condition a>b && a>c then not require to compare b>a, think on this

Reply:
Take reference of below code

https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/blob/main/src/sandesh/Assignment_4/NumberGame.java